scripts/trigger-manual-deploy.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+echo "Starting Auto Manual Start workflows ..."
+github_token="$1"
+project="$2"
+
+project=$(echo "$project" | tr '[:lower:]' '[:upper:]')
+if [[ $project == "SS" ]]; then
+  project="SDS"
+fi
+
+# environment list: sbox, ptlsbox, ithc, ptl, stg, demo, test, dev
+environment="$3"
+
```

Comment:
We should only be running this against a list of environments that we can add too - otherwise time can be wasted in builds